docs/source/rst/api/ML/core/lightning.rst

```diff
@@ -0,0 +1,15 @@
+Lightning Components
+================================
+
+.. automodule:: InnerEye.ML.lightning_base
```

Comment:
Another way of doing this is adding these to the modules' docstrings: https://github.com/microsoft/hi-ml/blob/main/hi-ml-multimodal/src/health_multimodal/image/__init__.py

But I guess this is fine too, although more work and chance of error.